ai2thor/tests/data/metadata-schema.json

```diff
@@ -521,9 +521,6 @@
         "colorBounds": {
             "type": "null"
         },
-        "reachablePositions": {
-            "type": "array"
-        },
```

Comment:
`"reachablePositions"` now no longer appears as a key in the metadata. However, it will silently alias to `actionReturn` if "GetReachablePositions" is the last action.